FoodFriends/app/src/main/AndroidManifest.xml

```diff
@@ -59,7 +59,8 @@
         </activity>
         <activity
```

Comment:
Is the app gonna be able to handle landscape view? Might be good to force portrait mode :)

Reply:
Set portrait for every screen except for Map Activity!